src/main/java/duke/command/AddCommand.java

```diff
@@ -0,0 +1,198 @@
+package duke.command;
```

Comment:
I notice a lot of repeated code segments in this class! might wanna consider moving some of the date formatting stuffz into another class!